frontend/common/types/responses.ts

```diff
@@ -466,7 +467,7 @@ export type Res = {
   rolePermission: PagedResponse<UserPermission>
   projectFlags: PagedResponse<ProjectFlag>
   projectFlag: ProjectFlag
-  identityFeatureStates: IdentityFeatureState[]
+  identityFeatureStatesAll: IdentityFeatureState[]
```

Comment:
Im a bit confused, why was this renamed?

Reply:
I think this name is clearer to refer to the endpoints: `/api/v1/environments/{environment_api_key}/edge-identities/{edge_identity_identity_uuid}/edge-featurestates/all/` and `/api/v1/environments/{environment_api_key}/identities/{identity_pk}/featurestates/all/`